### ISSUE TITLE
Top Navigation

### ISSUE THREAD

**Issue by brian-frederick** [brian-frederick/HPNC-Umbraco-Site#9]
Bar across the top
Options should be: Our Story, Calendar, Programs, Register, Space Rental, Volunteer, Staff, Donate (keep in Orange)
Programs drop down: Early Childhood, Elementary, Mentoring, Athletics
We may want to be able to edit which pages show up in this drop down, would that be possible?
Register should take user directly to HPNC proclass registration site (https://reg126.imperisoft.com/hpnc/Search/Registration.aspx)
After Donate, could we include icon links to our Social Media pages? 
Ex.  but gray?
Facebook: https://www.facebook.com/HydeParkNC; Instagram: https://www.instagram.com/hpnclub/; twitter: https://twitter.com/hpnclub

**Comment by brian-frederick** [comment]
@vemigh are we removing board from the dropdown? 

also, we could save ourselves some time and complexity if we moved the "Register" link to the end of the list, next to Donate. Would that be alright? 

The order of options in the dropdown and whether or not they are appear are dynamic and can be controlled in the CMS. 

Also, I feel like the social links might clutter our header a bit. What about the footer for these?

**Comment by vemigh** [comment]
Yes, so the link to that page would only be on that bottom section. Or you could also get to it from the newly created landing page for "Our Story"

Sure!

Great!

Okay, let's try that

**Comment by kanshemeza** [comment]
I can work on this.

**Comment by brian-frederick** [comment]
SUMMARY
In footer
- directly below annual report link, add logos (which link to corresponding Social Media accts) for facebook, instagram, twitter. All three of these should be side by side on one one line. use the same color as the surrounding text for color of the logos. 
 https://www.facebook.com/HydeParkNC; 
https://www.instagram.com/hpnclub/; 
twitter: https://twitter.com/hpnclub

@kanshemeza want to take a crack at this?

**Comment by brian-frederick** [comment]
the change to the header menu to add the Register option will now be handled in 
https://github.com/brian-frederick/HPNC-Umbraco-Site/issues/32
See directly above for details of this task

**Comment by brian-frederick** [comment]
hey @NickGardner do you have any advice on where to add these social links in the footer?

**Comment by brian-frederick** [comment]
@dakers2 this one's ready for dev if you want to take a look and get some social media icons for us.
